### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by schnogz**
## Description
Fixes bugs 710, 739, 740, 947

## Change Type
Bug Fix

## Testing Steps
See bug details.

## PR Creator Checklist
- [x] Code compiles correctly
- [x] No lint issues exist (verified via `yarn lint`)
- [x] All unit tests pass (verified via `yarn test`)
- [x] Updated `README.md` and other documentation (if necessary)

## PR Reviewer Checklist
- [ ] Change validated and application spot checked
- [ ] Code styles and best practices met
- [ ] Code is readable and commented when necessary

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js [hunk 1]
```diff
@@ -216,6 +216,7 @@ export default ({ api, coreSagas }) => {
     } catch (e) {
       yield put(actions.logs.logErrorMessage(logLocation, 'remindGuid', e))
       if (e.message === 'Captcha Code Incorrect') {
+        yield put(actions.core.data.misc.fetchCaptcha())
```
Comment (salomegeo) [inline packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js]: ah, does this refresh captcha? great!

File: packages/blockchain-wallet-v4-frontend/src/modals/AddBitcoinWallet/template.js [hunk 2]
```diff
@@ -22,7 +22,9 @@ const AddBitcoinWallet = (props) => {
   const { position, close, submitting, invalid, wallets, ...rest } = props
   const { onSubmit } = rest
 
-  const unique = (props) => map(prop('label'), wallets).indexOf(props) > -1 ? 'Wallet name is already taken.' : undefined
+  const unique = (value) => {
```
Comment (salomegeo) [inline packages/blockchain-wallet-v4-frontend/src/modals/AddBitcoinWallet/template.js]: what does this fix?

Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/modals/AddBitcoinWallet/template.js]: this file should not have been committed. i will uncommit.

File: packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js (hunk 1 above)
Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js]: yup!

**Pull request merged**